sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/throughputControl/controller/IThroughputController.java

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.implementation.throughputControl.controller;
+
+import com.azure.cosmos.implementation.RxDocumentServiceRequest;
+import reactor.core.publisher.Mono;
+
+public interface IThroughputController {
```

Comment:
Because of the `close()` API - I suggest we extend `AutoCloseable` interface.

Reply:
Discussed offline, will not implement the AutoCloseable.